<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogNewDownload</class>
 <widget class="QDialog" name="DialogNewDownload">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Download</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="jobLabel">
       <property name="text">
        <string>&amp;Job Name</string>
       </property>
       <property name="buddy">
        <cstring>jobLine</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="jobLine"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>File</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="metafileLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>Meta &amp;File</string>
          </property>
          <property name="buddy">
           <cstring>metafileLine</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="metafileLine"/>
        </item>
        <item>
         <widget class="QDialogButtonBox" name="metafileOpen">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="standardButtons">
           <set>QDialogButtonBox::Open</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="destinationLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>&amp;Destination</string>
          </property>
          <property name="buddy">
           <cstring>destinationLine</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="destinationLine"/>
        </item>
        <item>
         <widget class="QDialogButtonBox" name="destinationOpen">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="standardButtons">
           <set>QDialogButtonBox::Open</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="tempDirLabel">
          <property name="text">
           <string>&amp;Temp. Dir.</string>
          </property>
          <property name="buddy">
           <cstring>tempDirLine</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="tempDirLine"/>
        </item>
        <item>
         <widget class="QDialogButtonBox" name="tempDirOpen">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="standardButtons">
           <set>QDialogButtonBox::Open</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Security</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="decryptCheck">
        <property name="text">
         <string>de&amp;crypt</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="passwordLabel">
          <property name="text">
           <string>&amp;Password</string>
          </property>
          <property name="buddy">
           <cstring>passwordLine</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="passwordLine">
          <property name="text">
           <string/>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="showPWcheck">
        <property name="text">
         <string>&amp;show Password</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>104</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>decryptCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>passwordLabel</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>52</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>decryptCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>passwordLine</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>decryptCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>showPWcheck</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>71</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
